WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -647,14 +647,9 @@ private void LogCoinJoinSummary(ImmutableArray<AliceClient> registeredAliceClien
 
 		Logger.LogDebug($"{nameof(allowedNonPrivateCoins)}: {allowedNonPrivateCoins.Count} coins, valued at {Money.Satoshis(allowedNonPrivateCoins.Sum(x => x.Amount)).ToString(false, true)} BTC.");
 
-		// How many inputs do we want to provide to the mix?
-		var utxoCount = allowedNonPrivateCoins.Count + privateCoins.Length;
-		var minUtxoCountTarget = (int)((allowedNonPrivateCoins.Sum(x => x.Amount) + privateCoins.Sum(x => x.Amount)) / liquidityClue.Satoshi);
-		Logger.LogDebug($"Derived {nameof(minUtxoCountTarget)} from balance and expected liquidity: {minUtxoCountTarget}.");
-
 		int inputCount = Math.Min(
 			privateCoins.Length + allowedNonPrivateCoins.Count,
-			consolidationMode ? MaxInputsRegistrableByWallet : GetInputTarget(utxoCount, minUtxoCountTarget, rnd));
```

Comment:
Note that these things got removed because the implementation down aims to address the same issue, just in a more intelligent way.